Makefile

```diff
@@ -7,12 +7,28 @@
 #  Authors: anita.shekar@ibm.com, sandy.kaur@ibm.com
 #
 
+# MThis is the Makefile for acadia-platform. Based on from genctl/Acadia-service-workspace
```

Comment:
I don't think we need this reference here.

Reply:
removed